python/capypdf.py

```diff
@@ -389,6 +390,9 @@ def cmd_c(self, x1, y1, x2, y2, x3, y3):
     def cmd_cm(self, m1, m2, m3, m4, m5, m6):
         check_error(libfile.capy_dc_cmd_cm(self, m1, m2, m3, m4, m5, m6))
 
+    def cmd_d(self, array, phase):
+        check_error(libfile.capy_dc_cmd_d(self, ctypes.pointer(array), len(array), phrase)
```

Comment:
What happens if you pass an array of integers instead of doubles? Or mixed types? If CPython does not do a proper full conversion then you have a bad case of UB on your hands. 

I did something fancier in e.g. `set_icc` but if this actually works then all of that is unnecessary.